adagrams/game.py

```diff
@@ -1,11 +1,105 @@
+import random
+from webbrowser import get
```

Comment:
Looks like this import is unused so you can delete it. Perhaps it was accidentally brought in by VSCode. There should be an option to turn off auto completion for imports to avoid bringing in stray, unused imports